teaching_platform/lessons/exceptions.py

```diff
@@ -0,0 +1,63 @@
+import logging
```

Comment:
Exception handling should not be tied with logging. I suggest moving all logging from exceptions constructors into another place. The best place would be `except` in controller file.

Please apply this comment to all exceptions files in this PR.

Reply:
Done